src/components/document/document-content.tsx

```diff
@@ -404,4 +428,35 @@ export class DocumentContentComponent extends BaseComponent<IProps, IState> {
     }
   }
 
+  private scrollToSection(sectionId: string | null | undefined ) {
+    if (!sectionId || !this.domElement) {
+      return false;
+    }
+
+    const sectionElementTop = this.findSectionElementTop(this.domElement, sectionId);
+    if (sectionElementTop !== undefined) {
```

Comment:
I think we may have had this conversation before, but as a stylistic matter I prefer the `!= null` form to `!== null` or `!== undefined`. To me it reads simpler -- when I see one of the other forms I find myself wondering whether it makes sense for the the other (null in this case) to be handled by the if or the else clause. Not a strong preference, however.

Reply:
`!= null` only catches `null` and `undefined`, not any other falsy values like 0.